salt/utils/gitfs.py

```diff
@@ -1599,9 +1599,15 @@ def _traverse(tree, blobs, prefix):
             _traverse(tree, blobs, self.root(tgt_env))
         add_mountpoint = lambda path: salt.utils.path_join(self.mountpoint(tgt_env), path)
         for repo_path in blobs.get('files', []):
-            files.add(add_mountpoint(relpath(repo_path)))
+            if not salt.utils.is_windows():
+                files.add(add_mountpoint(relpath(repo_path)))
+            else:
+                files.add('/'.join(repo_path.replace('.:\\', '', 1).split(os.sep)))
```

Comment:
I'm not sure this is the correct fix (FWIW, I'm also not sure it's not). I'd be interested in knowing what led to ``r'.:\\'`` being put into the file list.

Can you share with me how you got pygit2 working in Windows? I haven't tried to get it running there due to libgit2 needing to be available, and my general deficiency in Windows knowledge when compared to Linux.